src/Common/Version.props

```diff
@@ -16,4 +16,4 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
   </PropertyGroup>
-</Project>
+</Project>
```

Comment:
EOL fixes

Reply:
This gets overwritten by the roll tool that we have, not sure this change is relevant.